### ISSUE TITLE
[Quality of life] Move certain extensions to more appropriate top-level menus

### ISSUE THREAD

**Issue by rbmntjs** [FontBureau/fbOpenTools#11]
I think it makes sense to let some extensions live in more specialised top-level menus, rather than all under Extensions. In particular, AdjustMetrics, GlyphSelect, ItalicBowtie and OverlayUFO could benefit from a more specific menu, rather than grouping in a catch-all.

**Comment by djrrb** [comment]
Yeah I do appreciate how "Batch" is in the File menu, and "Change Suffixes" is in the edit menu. 

However, I hesitate to dictate the locations of these extensions, and to change these after they have been in the generic Extensions menu for so long. Do you have specific locations in mind for them? (Glyph Select would make sense under the Glyph menu, but the others I'm not so sure).

Perhaps the solution is an "Extension Location" extension, which allows users to assign custom locations to any extension?

**Comment by rbmntjs** [comment]
I understand the concern of moving them after all this time. Your examples are exactly the ones I love for their placement.

My thinking for AdjustMetrics would be in Font, but the others are a bit less obvious. Italic Bowtie could be Edit, but, this really would be served by a set of editorial ideas about what these menus mean to people. I, uh… I didn’t come into this issue tracker to supply answers, clearly. But this sort of discussion is important in an ecosystem where a user may have dozens of extensions. So why not start here?

**Comment by djrrb** [comment]
> I, uh… I didn’t come into this issue tracker to supply answers, clearly.

;-D

> But this sort of discussion is important in an ecosystem where a user may have dozens of extensions. So why not start here?

Agreed. This is why I'm wondering whether a menu customizer extension might be a nice way to move forward, thinking about the big picture (not that I necessarily have the time or know-how to build that). 

Recently I have used the Extensions menu less and less, finding it easier to locate the scripts I need in a finder window and simply run them from there.

I will run this by other users I know and see what they think, and feel free to invite others here to comment. I'd like to have some sort of consensus before making changes to any of the fbTools extensions, especially since I'm not actively changing them.

**Comment by rbmntjs** [comment]
You might enjoy one of my favourite MacOS shortcuts, Command-Shift-? – in-app menu item search. But while it solves direct findability, it doesn’t improve sorting or categorisation. Anyway, that’s just an aside. Menu customisation might be the answer.
